packages/icons/src/library/comment-content.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { Path, SVG } from '@wordpress/primitives';
+
+const commentContent = (
+	<SVG viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+		<Path
+			fill-rule="evenodd"
```

Comment:
You have to change these props to `fillRule` and `clipRule` as `Path` is a React component. This way it produces React `Invalid DOM property` warnings.

Reply:
You're completely right. Thanks for noticing it. :+1: